projects/laji/src/app/shared/service/laji-api.service.ts

```diff
@@ -201,6 +201,15 @@ export namespace LajiApi {
 
 }
 
+export const removeUndefinedFromObject = <T extends object>(obj: T): T => {
+  return (Object.keys(obj) as (keyof T)[]).reduce((cumulative, current) => {
+    if (typeof obj[current] !== 'undefined') {
+      cumulative[current] = obj[current];
+    }
+    return cumulative;
+  }, {} as T);
+};
+
```

Comment:
Hmm, how about moving this to `util.service.ts`? It contains more similar utils functions, and could be more easily found by other developers.